### ISSUE TITLE
Unknown top-level box type imda

### ISSUE THREAD

**Issue by burak-kara** [gpac/gpac#2291]
Hi,

I tried to extract the box info of a segment using MP4Box with `-diso` option. I got `[iso file] Unknown top-level box type imda
` and `Error opening file <file_name>.mp4: IsoMedia File is truncated`.

Windows 64 bits
```
> MP4Box -version 
MP4Box - GPAC version 2.1-DEV-rev426-g3b6f2436-master
```

the sample video: https://www.mediafire.com/file/ungdop0ywxrenvk/omafv2.0.base.1.mp4/file

**Comment by Murmur** [comment]
Interesting file, it has only two atom tables.
```
styp (Segment Type Box), size=36
imda (Image Data Box), size=68396
```

**Comment by burak-kara** [comment]
> Interesting file, it has only two atom tables.
> 
> ```
> styp (Segment Type Box), size=36
> imda (Image Data Box), size=68396
> ```

yes, it is on purpose. It is one of the segment files of OMAF v2 content created using https://github.com/nokiatech/omaf. 

The boxes used in OMAF v2 were standardized in https://www.iso.org/standard/79881.html.

I am able to get box info using http://mp4parser.com/ (which is a really nice web tool), but mp4box on my local gives the error above.

**Comment by jeanlf** [comment]
> imda (Image Data Box), size=68396

it should be 'identified media data box', it's basically a mdat with an ID. We don't support that yet in gpac, I would need a regular file or fragment for that, but this one is just data (no moof, no moov) so it's impossible to parse alone.

I'll add the basic parse/dump asap.

**Comment by rbouqueau** [comment]
cf cb8ae46c82d49feed25d0feae0f84325a163c40c
